src/interfaces.ts

```diff
@@ -209,6 +206,7 @@ export type AusPostResponse = {
 export type UsersResponse = {
     userId: string;
     email: string;
+    createdAt: string;
```

Comment:
Not a time based class?

Reply:
`[
    {
        "userId": "09ebefb5-1e2d-4047-8856-0227c9de5ae4",
        "email": "dev@dltx.io",
        "createdAt": "2022-06-27T23:41:31.515Z"
    },
    {
        "userId": "d9abeaf5-50c2-485e-b645-b8557f4d47f4",
        "email": "b6e7543282bbf0627fdc11e649d0be871965fe87b16d0f3ebea39f82ed47ea62",
        "createdAt": "2022-06-30T11:46:01.362Z"
    },
    {
        "userId": "b9625262-0a14-497c-8b07-cd0c4a7d89a9",
        "email": "5b6416265295b815b20e5f8a28c7372a1fa94f3ab30cd6a9944244098eda90ab",
        "createdAt": "2022-07-01T01:06:01.973Z"
    }
]`

I think its good enough